firefox-ios/Client/Frontend/Onboarding/Views/TermsOfServiceViewController.swift

```diff
@@ -156,10 +165,10 @@ class TermsOfServiceViewController: UIViewController,
         ])
     }
 
-    // TODO: FXIOS-10347 Firefox iOS: Manage Privacy Preferences during Onboarding
-    private func setupAgreementTextView(with title: String, linkTitle: String, and a11yId: String) {
+    private func setupAgreementTextView(with title: String, linkTitle: String, linkType: LinkType, and a11yId: String) {
         let agreementLabel: UILabel = .build()
         agreementLabel.accessibilityIdentifier = a11yId
+        agreementLabel.tag = linkType.rawValue
```

Comment:
I don't think it's a good idea using a tag here. There is other ways to getting and displaying the correct link. I would suggest having a method for each link type which the gesture recognizer is calling. The terms of service and privacy link method can then call a `presentLink` method with the correct url.